hmvec/tinker.py

```diff
@@ -58,7 +59,7 @@ def f_nu(nu,zs,delta=200.,norm_consistency=True,
     gamma = gamma0 * (1+zs)**(-0.01)
     unnormalized = (1. + (beta*nu)**(-2.*phi))*(nu**(2*eta))*np.exp(-gamma*nu**2./2.)
     if norm_consistency:
-        izs,ialphas = np.loadtxt("alpha_consistency.txt",unpack=True) # FIXME: hardcoded
+        izs,ialphas = np.loadtxt("/Users/wcoulton/Downloads/Software/hmvec/data/alpha_consistency.txt",unpack=True) # FIXME: hardcoded
```

Comment:
Please revert this. Also, we should be testing with Sheth-Torman (which gives the best fit to halofit), so tinker is not really used anyway.

Reply:
Done.